### PR TITLE
Keep aligned with our released stable package

### PR DESCRIPTION
**Pull request by xboxeer**
there is one Stackoverflow question https://stackoverflow.com/questions/71061498/net-azure-sdk-resource-manager-nuget-package-stable-release asking which version of SDK to use, so it is better we update our docs to include the stable version of new SDK

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/azure/sdk/resource-management.md [hunk 1]
```diff
@@ -30,10 +30,10 @@ Install the Azure Resources management packages for .NET with [NuGet](https://ww
 For example:
 
 ```PowerShell
-Install-Package Azure.ResourceManager -Version 1.0.0-beta.5
+Install-Package Azure.ResourceManager -Version 1.0.0
```
Comment (ArthurMa1978) [inline docs/azure/sdk/resource-management.md:33]: now we consider using donetcli,

```dotnetcli
dotnet add package Azure.ResourceManager
dotnet add package Azure.ResourceManager.Resources
dotnet add package Azure.ResourceManager.Compute  --prerelease
```

**Comment by IEvangelist** [comment]
Tagging @scottaddie for review. Scott, are these updates accurate?

File: docs/azure/sdk/resource-management.md [hunk 2]
```diff
@@ -29,11 +29,20 @@ Install the Azure Resources management packages for .NET with [NuGet](https://ww
 
 For example:
 
+# [PowerShell](#tab/PowerShell)
```
Comment (IEvangelist) [inline docs/azure/sdk/resource-management.md:32]:
```suggestion
# [PowerShell](#tab/PowerShell)
```

File: docs/azure/sdk/resource-management.md [hunk 3]
```diff
@@ -29,11 +29,20 @@ Install the Azure Resources management packages for .NET with [NuGet](https://ww
...
 ```PowerShell
-Install-Package Azure.ResourceManager -Version 1.0.0-beta.5
+Install-Package Azure.ResourceManager -Version 1.0.0
+Install-Package Azure.ResourceManager.Resources -Version
 Install-Package Azure.ResourceManager.Compute -Version 1.0.0-beta.3
 Install-Package Azure.ResourceManager.Network -Version 1.0.0-beta.3
-Install-Package Azure.ResourceManager.Resources -Version 1.0.0-beta.3
+```
+
+# [dotnet cli](#tab/dotnetcli)
```
Comment (IEvangelist) [inline docs/azure/sdk/resource-management.md:40]:
```suggestion
# [dotnet cli](#tab/dotnetcli)
```

**Review by IEvangelist — COMMENTED** [review]
This looks good, thank you. We'll :shipit: when you make the suggested changes.

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**